docs/src/tools.md

```diff
@@ -26,15 +26,10 @@ Copy command can also be used to compare two mbtiles files and generate a diff.
 mbtiles copy src_file.mbtiles diff_file.mbtiles --force-simple --diff-with-file modified_file.mbtiles
 ```
 
-The `diff_file.mbtiles` can be applied to the `src_file.mbtiles` elsewhere to avoid copying/transmitting the entire modified dataset.
-
-One way to apply the diff is to use the `sqlite3` command line tool directly. Here, we assume that the `src_file.mbtiles` is in the simple tables format, and that the `diff_file.mbtiles` is the output of the `mbtiles copy` command above. This SQL will delete all tiles from `src_file.mbtiles` that are set to `NULL` in `diff_file.mbtiles`, and then insert or update all new tiles from `diff_file.mbtiles` into `src_file.mbtiles`. The name of the diff file is passed as a query parameter to the sqlite3 command line tool, and then used in the SQL statements.
-
+### apply-diff
+Apply the diff file generated from `copy` command above to an mbtiles file. The diff file can be applied to the `src_file.mbtiles` elsewhere, to avoid copying/transmitting the entire modified dataset.
 ```shell
-sqlite3 src_file.mbtiles \
```

Comment:
This script should be preserved somewhere but not here, since the `apply-diff` command replaces it

Reply:
I would prefer to keep both of them here. This is documentation, so the more thorough it is, the better